wolfcrypt/src/dh.c

```diff
@@ -1793,6 +1798,8 @@ static int _ffc_pairwise_consistency_test(DhKey* key,
 
     if (key == NULL || pub == NULL || priv == NULL)
         return BAD_FUNC_ARG;
+    if (mp_iseven(&key->p) == MP_YES)
```

Comment:
I see these checks for even added in ECC, DH and RSA. These are good and trap early an invalid value. Should these be in a separate commit or PR as they are not specifically related to the P521 addition?

Reply:
Don't remember how this came up now!
https://github.com/wolfSSL/wolfssl/pull/4882